helm/inji-verify-ui/values.yaml

```diff
@@ -51,7 +51,7 @@ service:
 
 image:
   registry: docker.io
-  repository: mosipdev/inji-verify
+  repository: mosipqa/inji-verify-ui
```

Comment:
Why are we using QA images ?

Reply:
QA images will be stable so it preferred to use them in helm charts